extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementPageQuery.java

```diff
@@ -21,10 +24,16 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
 @Schema(description = "Filters for querying a Contract Contract Agreement Page")
 public class ContractAgreementPageQuery {
     @Schema(
         description = "Optionally filter the resulting contract agreements by their termination status.",
         requiredMode = Schema.RequiredMode.NOT_REQUIRED)
     private ContractTerminationStatus terminationStatus;
+
+    @JsonIgnore
+    public boolean isEmpty() {
+        return terminationStatus == null;
+    }
```

Comment:
we don't want to fill our DTO model with these kinds of utility methods, as we don't want to have to start "testing" our API model. We want to keep them as POJOs

please move this elsewhere